base/applications/mspaint/common.h

```diff
@@ -10,6 +10,22 @@
 
```

Comment:
(I cannot comment directly on the line with the github interface)
The "purpose" of this file can be adjusted to mention that there are also commonly used definitions too.

Reply:
OK.